source/platforms/google-colab.md

```diff
@@ -0,0 +1,44 @@
+# RAPIDS on Google CoLab
+
+## 1. Set the Runtime
+
+In [Google Colab](https://colab.research.google.com/), click the `Runtime` dropdown at the top toolbar, then `Change Runtime Type`
+
+![Screenshot of create runtime and runtime type](../../images/googlecolab-select-runtime-type.png)
```

Comment:
These screenshots aren't displaying right in the build preview. Could you check the paths?

https://rapids-deployment--180.org.readthedocs.build/en/180/platforms/google-colab.html